environments/ithc/backend_lb_config.yaml

```diff
@@ -266,10 +266,6 @@ gateways:
       - product: civil
         component: service
 
-      # RPTS
```

Comment:
Not related to this bit but, but see comment
[environments/test/test.tfvars](https://github.com/hmcts/azure-platform-terraform/blob/02ad1551a560c6b8cedf4b077f8a6ec3006f49c6/environments/test/test.tfvars#L392) 
this may also need removing

Reply:
removed